### ISSUE TITLE
Als medewerker gebiedsontwikkeling wil ik op basis van bepaalde kenmerken van appartements-adressen zoals bouwjaar en energielabel verenigingen van eigenaren kunnen zoeken

### ISSUE THREAD

**Issue by CathyDingemanse** [VNG-Realisatie/Haal-Centraal-BAG-bevragen#306]
zodat ik eigenaren kan aanschrijven in het kader van de energietransitie en energiemaatregelen kan communiceren.

**Acceptatiecriteria**
- [ ]
- [ ]

**Definition of done**
- [ ] functionele specificatie
- [ ] Open API specificatie
- [ ] gegenereerde code
- [ ] testgevallen

**Comment by strijm** [comment]
@CathyDingemanse Hier wordt meer gevraagd dan de BAG kan leveren, is het mogelijk deze story concreet te maken voor de BAG?

**Comment by CathyDingemanse** [comment]
zie #380

**Comment by CathyDingemanse** [comment]
@ Nicole, het is niet zozeer de bedoeling om alleen verblijfsobjecten te selecteren met een woonfunctie. Het is wel de bedoeling verblijfsobjecten te selecteren een bepaald bouwjaar/ bouwperiode/ (energielabel). Of is dat een selectie op basis van bouwjaar pand? 
Dan is het vervolgens de bedoeling om van deze verblijfsobjecten (in panden met bouwperiode xxx) de VVE te vinden in de BRK, of de eigenaar van het pand/verblijfsobjecten als er sprake is van 1 eigenaar.

**Comment by NicoleKortoomsBAG** [comment]
@CathyDingemanse, het klopt dat het bouwjaar alleen gekoppeld is aan het pand. Het pand zou gevonden kunnen worden met een contour en een periode te kiezen. Vervolgens zouden de verblijfsobjecten (ado's) in een pand kunnen worden gegeven. Is het ADO waar in de BRK op wordt gezocht om de VVE of de eigenaar te vinden?

**Comment by CathyDingemanse** [comment]
Ja, ADO of bijbehorende nummeraanduiding ligt het meest voor de hand.
